lib/utils/device/progress.ts

```diff
@@ -18,7 +18,7 @@ import { Device } from 'balena-sdk';
 
 export const getDeviceOsProgress = (device: Device) => {
 	if (!device.is_online) {
-		return 0;
+		return null;
```

Comment:
OS Update progress bar was resetting to 0% at end of process, as device went offline during reboot.  I think null is more appropriate in these cases.